wsapi.go

```diff
@@ -351,6 +351,9 @@ func (s *Session) UpdateStreamingStatus(idle int, game string, url string) (err
 			URL:  url,
 		}
 	}
+	if game == "" {
+		usd.Game = nil
```

Comment:
This shouldn't be needed, the default is nil.

Reply:
This is definitely not the case, if I've sent "" without this change, the game has been left as it was previously set to.

Reply:
Hmmmmm, I think I see what's going on, this check should be in UpdateListeningStatus then, sorry for the back and forth on this one!

Reply:
This behaviour exhibits itself when setting an actual game too, which is why I put the check in there. I do agree that I probably need a similar check in UpdateListeningStatus as well, however.